### DISCUSSION TITLE
Emails can be sent as Plain Text

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9417] [Ideas]
> Created by: Ben Parizek (ben@barrelstrengthdesign.com) on 2015/03/12 23:19:32 +0000
> Votes at time of UserVoice import: 3

-----------------------------------

Currently, I don't believe there is a way to send Plain Text emails (only text/plain, not multipart/mixed with text) using Craft's Email Service.  As I understand it, if no HTML email is provided, Craft will take whatever it has as the plain text email, run it through a Markdown filter, and use that as the HTML email content.

I believe it is still necessary and desirable to send real plain text emails at times, and while I have no issues with Craft defaulting to HTML, I think we should have an option to send Plain Text as Plain Text without it being converted to HTML on the fly.

Technological improvements have made it much less necessary to worry about plain text but, in some environments, and with some audiences (.gov, .edu), getting caught in spam filters or firewalls is still an issue and choosing to send email in plain text to reduce the chance of getting picked off and increasing readability is desirable.

**Comment by j-greig** [comment]
Any update on this? Our client's emails are getting mangled by their PC :-(

**Comment by brandonkelly** [comment]
@j-greig Emails are sent with both plain text and HTML variants, and most email clients will give let you choose which one to see (though not in Mail.app since macOS 10.9 apparently).

Not opposed to adding a setting to force plain text, but I'm curious if we can fix the HTML variant. Can you tell me which email client & version they use?
